packages/gitmoji-changelog-core/src/fromGitFile.js

```diff
@@ -0,0 +1,46 @@
+const gitRawCommits = require('git-raw-commits')
+const splitLines = require('split-lines')
+const { promisify } = require('util')
+const gitSemverTags = require('git-semver-tags')
+const through = require('through2')
+const concat = require('concat-stream')
+
+const COMMIT_FORMAT = '%n%H%n%an%n%cI%n%s%n%b'
+
+function parseCommit(commit) {
+  const lines = splitLines(commit)
+  const [hash, author, date, subject, ...body] = lines.splice(
+    1,
+    lines.length - 2
+  )
+  return {
+    hash, author, date, subject, body,
+  }
+}
```

Comment:
This need to be unit tested.

Reply:
The core is unit tested using this client. It is not mandatory to merge this pull request?